testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeChangeDataFeedCompatibility.java

```diff
@@ -577,6 +578,74 @@ public void testThatCdfDoesntWorkWhenPropertyIsNotSet()
         assertThereIsNoCdfFileGenerated(tableName2, "change_data_feed_enabled = false");
     }
 
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, PROFILE_SPECIFIC_TESTS})
```

Comment:
add `DELTA_LAKE_OSS`

Just a hint, the product tests on delta lake oss run much faster locally.

Reply:
oh god I missed this one